README.md

```diff
@@ -8,6 +8,61 @@ Puppet configuration and environment management for the Grasshopper event engine
  * If you make changes to the backend code you will need to restart the app server. This can be done by ssh'ing into the client machine by running `service grasshopper restart`.
  * Even if you'd install all the components on your host OS, you would not be able to run the server as some of the npm modules are compiled during the provisioning step.
 
+### Dev Environment (e.g. on EC2)
+
+```
```

Comment:
Minor: could you take all of this out of the comments block and only use `this style of markup` for the lines to execute?